server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/BulkImportCacheCleaner.java

```diff
@@ -54,9 +54,13 @@ public void run() {
       for (Tablet tablet : server.getOnlineTablets().values()) {
         tablet.cleanupBulkLoadedFiles(tids);
       }
-    } catch (KeeperException | InterruptedException e) {
+    } catch (KeeperException e) {
       // we'll just clean it up again later
-      log.debug("Error reading bulk import live transactions {}", e);
+      log.debug("Error reading bulk import live transactions {}", tids, e);
+    } catch (InterruptedException e) {
+      // propagate the interrupt status.
+      Thread.currentThread().interrupt();
+      log.debug("Interrupted while reading bulk import live transactions {}", tids, e);
```

Comment:
Sorry should have considered this - but might want to consider bumping up the log level - we really shouldn't be seeing interrupts and if the keep reoccurring and are not being handled, then we should dig deeper.

Reply:
Bumped to warn in 307bef2